sdk/eventhub/event-hubs/karma.conf.js

```diff
@@ -48,7 +48,8 @@ module.exports = function(config) {
     envPreprocessor: [
       "EVENTHUB_CONNECTION_STRING",
       "EVENTHUB_NAME",
-      "IOTHUB_EH_COMPATIBLE_CONNECTION_STRING"
+      "IOTHUB_EH_COMPATIBLE_CONNECTION_STRING",
+      "TEST_TARGET"
```

Comment:
Make it TEST_MODE?

Reply:
The multiversion test helper makes some assumptions with `TEST_MODE` that causes issue with what we're trying to do. I'm using the multiversion test helper to facilitate testing mock and live versions of the service, not multiple API versions of the service.

This works fine if I don't set TEST_MODE at all, but things break if I set TEST_MODE to live because it assumes I want different behavior than just running my tests against the live service. So, that's why I made TEST_TARGET instead.